### DISCUSSION TITLE
Conditional element type field

### DISCUSSION THREAD

**Discussion by deanCompRSA** [vueform/vueform#337] [Questions]
I have this use case where I have a select. This select has two values yes or no which should conditionally display one of two fields. This would work for normal cases, however the issue is that both fields have the same name and the form breaks if there are two elements with the same name. How could I conditionally set the element type or any other better way to handle this?

https://stackblitz.com/edit/github-nzqvai-kip4bw?file=src%2FApp.vue,src%2Fstyle.css,vueform.config.ts

```
  <Vueform>
 <SelectElement
      name="fromOptions"
      :items="[
        'Yes',
        'No',
      ]"
    />
 <SelectElement
      name="myValue"
      :items="[
        '1',
        '2',
      ]",
      :conditions="[
        ['fromOptions', '==', 'Yes']
      ]"
    />
    <TextElement
      name="myValue"
      :conditions="[
        ['fromOptions', '==', 'No']
      ]"
    >
  </Vueform>
```

**Comment by adamberecz** [comment]
As you have to name the two elements differently I usually use eg. `_select` and `_text` suffix than transform the data on submit and load.
